tests/test_json_field_ids.py

```diff
@@ -0,0 +1,91 @@
+import copy
+import json
+from pathlib import Path
+
+files = Path("form_jsons/public").glob("*")
+forms_using_field_id = {}
+form_json_with_field_ids = []
+
+
+def get_form_jsons_with_field_ids():
+    form_json_with_field_ids = []
+    for file in files:
+        form_name = file.name
+        field_ids = []
+        with open(file) as jsonFile:
+            data = json.load(jsonFile)
+            for page in data["pages"]:
+                for component in page["components"]:
+                    field_id = component["name"]
+                    field_ids.append(field_id)
+
+        form_json_with_field_ids.append(
+            {"form_name": form_name, "field_ids": field_ids}
+        )
+
+    return form_json_with_field_ids
+
+
+def are_strings_unique(string_one, string_two):
+    if string_one == string_two:
+        return False
+    else:
+        return True
```

Comment:
Could we simplify this as follows?
```suggestion
    return string_one != string_two
```

Reply:
yes - good shout 💯